server/Arcadia.Assistant.Web/Controllers/EmployeesController.cs

```diff
@@ -75,6 +75,8 @@ public EmployeesController(IActorRefFactory actorSystem, ActorPathsBuilder paths
                         var employee = EmployeeModel.FromMetadata(x.Metadata);
                         var photo = await x.Actor.Ask<GetPhoto.Response>(GetPhoto.Instance, timeout, token);
                         employee.Photo = photo.Photo;
+                        employee.HoursCredit = 12;
```

Comment:
Did you forget to commit this property?

Reply:
no, the property is committed, name change didn't pass through though...

Reply:
fixed